README.md

```diff
@@ -22,6 +22,12 @@ On release, automated continuous integration tests run the pipeline on a [full-s
 
 The pipeline is built using [Nextflow](https://www.nextflow.io), a workflow tool to run tasks across multiple compute infrastructures in a very portable manner. It uses Docker/Singularity containers making installation trivial and results highly reproducible. The [Nextflow DSL2](https://www.nextflow.io/docs/latest/dsl2.html) implementation of this pipeline uses one container per process which makes it much easier to maintain and update software dependencies. Where possible, these processes have been submitted to and installed from [nf-core/modules](https://github.com/nf-core/modules) in order to make them available to all nf-core pipelines, and to everyone within the Nextflow community!
 
+## Online videos
+
+A short talk about the history, current status and functionality on offer in this pipeline was given by Harshil Patel ([@drpatelh](https://github.com/drpatelh)) on [8th February 2022](https://nf-co.re/events/2022/bytesize-32-nf-core-rnaseq) as part of the nf-core/bytesize series.
```

Comment:
Won't be good to also directly include the direct link to youtube?

Reply:
I wanted to point to the nf-core pages instead because it just brings users in contact with the website a little more and therefore brings their attention to other events and stuff we have there. The video is linked to Youtube there anyway?

Reply:
Yes it is linked there, fair enough